### ISSUE TITLE
add test suite to monaco-graphql

### ISSUE THREAD

**Issue by acao** [graphql/graphiql#1607]
our beloved `monaco-graphql` _could_ have a really nice test suite!

[`monaco-yaml`'s suite](https://github.com/pengx17/monaco-yaml/tree/master/test) provides a fantastic example

we can just start with something similar to the hover/completion/diagnostics suites. we don't need to worry about multiple documents or completion with multiple schemas any time soon

**Comment by faraz16iqbal** [comment]
Hey, I'm a recent starter and would be interested in knowing about this issue and working on it!

**Comment by acao** [comment]
great to hear, @faraz16iqbal , that would be great to have help with. have you worked with jest or any other test runner before?

**Comment by faraz16iqbal** [comment]
Nope, unfortunately I haven't. But I'll be more than thrilled to look and work into it!

**Comment by iamdarshshah** [comment]
Hey @acao 👋 

If you haven't assigned this issue to anyone yet, then I would like to know more about it and would be interested in taking up this issue.

Thanks

**Comment by acao** [comment]
@iamdarshshah sadly, it seems like they took their test suite away since I created this ticket a few months ago!

because of that, and changes to the architecture of the monaco-graphql plugin itself, I'm going to be introducing a suite soon, and I'll be sure to give you a good starting point to test very specific features

**Comment by iamdarshshah** [comment]
@acao Looking forward to it. Thanks 👍
